### PR TITLE
[RPI5] Support raspberry 5

### PR DESCRIPTION
**Pull request by mlohvynenko**

### CODE REVIEW

File: boards/rpi_5.conf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+CONFIG_FILE_SYSTEM_LITTLEFS=n
+
+CONFIG_FAT_FILESYSTEM_ELM=y
+CONFIG_FS_FATFS_MOUNT_MKFS=n
+CONFIG_FS_FATFS_EXFAT=y
+CONFIG_FS_FATFS_LBA64=y
+CONFIG_FS_FATFS_NUM_FILES=16
+
+CONFIG_FLASH=n
```
[the remark this answers is not in the document]
Comment (mlohvynenko) [reply boards/rpi_5.conf]: This  leads to the next error

```
/home/mykhailo_lohvynenko/projects/aos/aos_zephyr_sdk/zephyr/drivers/flash/flash_handlers.c:19:10: fatal error: syscalls/flash_read_mrsh.c: No such file or directory
   19 | #include <syscalls/flash_read_mrsh.c>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: west.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ manifest:
 
     - name: zephyr-optee-client
       remote: xen-troops
-      revision: "soafee_1.1_wip"
+      revision: "master"
```
[the remark this answers is not in the document]
Comment (mlohvynenko) [reply west.yml]: Xen team suggested to use this latest tag.
can you please clarify how I should negotiate it properly?

File: Kconfig [hunk 3]
```diff
@@ -109,12 +109,18 @@ config AOS_SOCKET_SERVER_PORT
 	depends on NATIVE_APPLICATION
 	default 30001
 
+config DOMD_ENABLE
```
Comment (al1img) [inline Kconfig:112]: Move DOMD related options togather with DOMU related options.

File: boards/rpi_5.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (mlohvynenko) [reply boards/rpi_5.conf]: build succeeded with -p always west param

File: src/bsp/mount_fatfs.c [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    ret = disk_access_ioctl(DISK_DRIVE_NAME, DISK_IOCTL_GET_SECTOR_SIZE, &block_size);
+    if (ret) {
+        printk("[fatfs] get sector size failed (%d)\n", ret);
+
+        return ret;
+    }
+
+    mp.mnt_point = CONFIG_AOS_DISK_MOUNT_POINT;
```
Comment (al1img) [inline src/bsp/mount_fatfs.c:62]: We should remove CONFIG_AOS_DISK_MOUNT_POINT option at all. fatfs should be mount to its own mount point as well as little fs on its own. In general case we may have few fatfs parts as well as few littlefs parts mounted.

File: boards/rpi_5.conf [hunk 5]
```diff
@@ -11,7 +11,7 @@ CONFIG_XEN_DOM0LESS_BOOT=y
 CONFIG_NOCACHE_MEMORY=y
 CONFIG_FS_MULTI_PARTITION=y
 
-CONFIG_FILE_SYSTEM_LITTLEFS=n
+CONFIG_FILE_SYSTEM_LITTLEFS=y
```
Comment (al1img) [inline boards/rpi_5.conf:14]: drop this commit.

File: west.yml [hunk 6]
```diff
@@ -41,7 +41,7 @@ manifest:
 
     - name: zephyr-optee-client
       remote: xen-troops
-      revision: "soafee_1.1_wip"
+      revision: "v0.1.1"
```
Comment (al1img) [inline west.yml:44]: Put master here. We will fix revision when create release

**Pull request merged**